CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Thank you!
 # 0.18.21
```

Comment:
thought: we're gonna have to move it to 0.18.22

Reply:
that seems correct, as `0.18.21` is already published

Reply:
Updated